File: Data_Structure/javascript/1764.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
const [n,m] = input[0].split(' ').map(Number);
let result = new Map();
let i;
for ( i = 1 ; i<=n ; i++){
    result.set(input[i],1);
}
for (; i<input.length-1 ; i++){
    if (result.has(input[i])){
        result.set(input[i],2);
    }
}
let cnt=0;
let answer = [];
for (let s of result.keys()){
  if (result.get(s) === 2){
    answer.push(s);
    cnt++;
  }
}

console.log(cnt);
console.log(answer.sort().join("\n"));

File: Data_Structure/python/1764.py
n,m = map(int,input().split())
dic = {}
for _ in range(n):
  dic[input()] = 1;
for _ in range(m):
  s = input()
  if s in dic:
    dic[s] = 2;
cnt = 0
answer = []
for s in dic.keys():
  if dic[s] ==2:
    cnt+=1
    answer.append(s)
answer.sort()
print(cnt)
print("\n".join(answer))